norddrop/src/device/mod.rs

```diff
@@ -120,15 +120,15 @@ impl NordDropFFI {
         let ed = self.event_dispatcher.clone();
         let event_logger = self.logger.clone();
         let event_storage = storage.clone();
-        let (tx, mut rx) = mpsc::unbounded_channel();
+        let (tx, mut rx) = mpsc::unbounded_channel::<(Event, u64)>();
```

Comment:
Very minor: I think `(Event, SystemTime)` would state the intent better. But you can skip this

Reply:
Can't say i really like where the acquisition of the timestamp would move with this change, will choose to skip this :)

Reply:
just tagging along - `SystemTime` would definitely improve the semantics of this

Reply:
It was wrong wording by me, i meant the calculation of the timestamp instead of the acquisition

i've changed it to `SystemTime` though